script/Ciel/Day 13/13_01_01_CIEL13_5.txt

```diff
@@ -206,92 +235,115 @@ On top of that, there's also the vampire standing before us, Noel. Despite once
+ It's hilarious, it feels like I've got a cheat code!"
 }
 [sha:c44b8b9beeee05ab9151a72e0bca62e0a4cab281]{
 -- Page 529, Offset 34799.
 -- ……死者の数はまだ増えていく。
-...The number of Dead continues to grow.
+More and more Dead keep showing up...
 }
 [sha:9448248db2bad31ec2d2ee02a6567d22d1967bb1]{
 -- Page 529, Offset 34800.
 -- 数分を待たず、俺たちは完全に包囲されてしまった。
-In the span of mere minutes, we've been completely surrounded on all sides.
+It hasn't even been three minutes,
+ and we're already completely surrounded.
 }
 [sha:c8428ca1e014bb7733bc5e3802ae7c9234320db3]{
 -- Page 530, Offset 34801.
 -- 「……よく分かりました。
-"...I get it now.
+"I see what's happening here.
 }
 [sha:99800349fca552922262cabc15c33b68f68c8b4b]{
 -- Page 530, Offset 34802.
 -- 経緯はどうあれ、貴女は吸血鬼になった。吸血鬼として生き延びる為に、<代|わ><行|た><者|し>を殺しに来たのですね」
-Regardless of how it happened, the fact is that you've become a vampire. As such, in order to live on, you've come to dispose of me, an Executor."
+Regardless of how it happened, the fact is that you've become a vampire.
+ As such, in order to keep yourself alive, you've come to kill the remaining Executor: me."
 }
 [sha:1ebf3f801fcdbeee89b89881e96cd588337bfaf1]{
 -- Page 531, Offset 34803.
 -- 痛みに顔を<歪|ゆが>ませる事なく、先輩は胸を貫いていた凶器を引き抜く。
-Without a single trace of her face contorting in pain, Senpai pulls the iron fence out of her chest.
+Senpai doesn't so much as flinch as she rips the iron spire out of her chest.
 }
 [sha:feeb27fc1743e47f3191296aabd54d451f6aa593]{
 -- Page 531, Offset 34804.
 -- その横顔は―――さっきまで散々見てきた、涙する一歩手前の、代行者としての顔だ。
-Just like a while ago, she has that same look on her face again―――the look of an Executor on the verge of tears.
+She has that look in her eyes again――the same one I've seen time and again at this point.
+ The look of an Executor who's on the verge of tears.
 }
 [sha:a5abc9eb9464faae94f0220ee871858d67e4ec29]{
 -- Page 532, Offset 34805.
 -- ……戦いが始まる。
-...The battle is about to begin.
+The battle is about to begin...
 }
 [sha:5304f37c8f7f1a63a0b31ef7e7f068a7b94d249d]{
 -- Page 532, Offset 34806.
 -- いや、単純明快な“討伐”が始まる。
-No, simply put, it's the 'subjugation' that's about to begin.
+No, let's call it what it is――the %{i}subjugation%{/i} is about to begin.
 }
 [sha:12ff12c38cbc265bde7f38dca02a02039d655f00]{
 -- Page 533, Offset 34807.
 -- シエル先輩とノエルの実力差は明らかだ。
-The difference in abilities between Ciel-senpai and Noel is evident.
+The difference in ability between Ciel-senpai and Noel is night and day.
 }
 [sha:6cb6f0ced91d612234410d3060bf78d4c62a99a2]{
 -- Page 533, Offset 34808.
 -- ノエルの体を覆う呪いの規模は、人間を遥かに凌駕している。以前の彼女から10倍や20倍では利かない性能の向上だろう。
-The curse surrounding Noel's body far surpasses that of a human being. She's gotten so much stronger that even if her abilities from when she was human were multiplied ten or twentyfold, her past self would be no match for the current her.
+With the curse enveloping her body, Noel far surpasses an ordinary human.
+ She's at least ten times, if not twenty times, stronger than she was before.
```

Comment:
Not sure I like that change because on first reading I thought it meant that it was too much for _Noel_ to handle,  and sure, she's not human, but it's still a little ambiguous. Are you able to pinpoint what exactly makes this ordering lore-interesting?

Reply:
That curses have "scales" to them, essentially. That was about it.